label_studio/users/models.py

```diff
@@ -199,6 +199,12 @@ def reset_token(self):
             token.delete()
         return Token.objects.create(user=self)
 
+    def soft_delete(self):
+        self.is_deleted = True
+        token = Token.objects.filter(user=self)
+        if token.exists():
+            token.delete()
```

Comment:
this can be `Token.objects.filter(user=self).delete()`, one fewer query

Reply:
good catch, might as well improve the piece of code I copied it from as well (the method above)😆